src/main/java/io/xpring/xrpl/model/XRPTransaction.java

```diff
@@ -201,7 +235,34 @@ static XRPTransaction from(Transaction transaction) {
       }
     }
 
+    // Transactions report their timestamps since the Ripple Epoch, which is 946,684,800 seconds after
+    // the unix epoch. Convert transaction's timestamp to a unix timestamp.
+    // See: https://xrpl.org/basic-data-types.html#specifying-time
+    Optional<Integer> timestamp = Optional.empty();
+    if (getTransactionResponse.hasDate()) {
+      Integer rippleTransactionDate = getTransactionResponse.getDate().getValue();
+      if (rippleTransactionDate != null) {
+        timestamp = Optional.of(rippleTransactionDate + 946684800);
+      }
+    }
+
+    Optional<String> deliveredAmount = Optional.empty();
+    if (getTransactionResponse.getMeta().hasDeliveredAmount()) {
+      CurrencyAmount currencyAmountDelivered = getTransactionResponse.getMeta().getDeliveredAmount().getValue();
+      switch (currencyAmountDelivered.getAmountCase()) {
+        case XRP_AMOUNT:
+          deliveredAmount = Optional.of(Long.toString(currencyAmountDelivered.getXrpAmount().getDrops()));
+          break;
+        case ISSUED_CURRENCY_AMOUNT:
+          deliveredAmount = Optional.of(Long.toString(currencyAmountDelivered.getXrpAmount().getDrops()));
```

Comment:
this line is the same as 254, assume you meant to get issuedCurrencyAmount? 

We may consider adding a test for this case?

Reply:
another very good catch.  I added two test cases to check this specifically...